### DISCUSSION TITLE
Mukurtu site in private mode

### DISCUSSION THREAD

**Discussion by MonicaGalassi** [MukurtuCMS/Mukurtu-CMS#29] [Feature Enhancement]
We get often asked if it's possible to have Mukurtu as a complete private site (homepage included) to be excluded from google search when the site is under construction

**Comment by michael-wynne-wsu** [comment]
That's something that might need to be more of a drupal/directory backend tool, but we'll look into a more user friendly option.

I'm thinking https://developers.google.com/search/docs/advanced/crawling/block-indexing is a good place to start in the meantime.

**Comment by rtrvrtg** [comment]
I've used the https://www.drupal.org/project/robots_noindex_nofollow module on quite a few Drupal 8+ installations. I don't know whether it should be part of the core Mukurtu distribution but perhaps it could be referred to in documentation as a useful module that users might want to have available.

**Reply by michael-wynne-wsu** [reply]
Great suggestions! I agree that it likely won't be in core, but definitely something we'll try to point to - perhaps a roundup of modules and tools for more private site management.
